File: Iniciante/1013.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        int valor1 = scanf.nextInt();
        int valor2 = scanf.nextInt();
        int valor3 = scanf.nextInt();

        if (valor1 > valor2 && valor1 > valor3) {
            System.out.printf("%d eh o maior\n", valor1);
        } else if (valor2 > valor3) {
            System.out.printf("%d eh o maior\n", valor2);
        } else {
            System.out.printf("%d eh o maior\n", valor3);
        }
    }
    
}

File: Iniciante/1013.py
valor1, valor2, valor3 = input().split(" ")
valor1 = int(valor1)
valor2 = int(valor2)
valor3 = int(valor3)

if valor1 > valor2 and valor1 > valor3:
	print(f'{valor1} eh o maior')
elif valor2 > valor3:
	print(f'{valor2} eh o maior')
else:
	print(f'{valor3} eh o maior')